src/Symfony/Component/Runtime/Runner/Symfony/HttpKernelRunner.php

```diff
@@ -24,13 +25,24 @@ class HttpKernelRunner implements RunnerInterface
     public function __construct(
         private readonly HttpKernelInterface $kernel,
         private readonly Request $request,
+        private readonly bool $debug = false,
     ) {
     }
 
     public function run(): int
     {
         $response = $this->kernel->handle($this->request);
-        $response->send();
+        $response->sendHeaders();
```

Comment:
Copy-pasting things here is easier and looks cleaner than somehow adding a "debug" logic in the Response class.